robyn/__init__.py

```diff
@@ -583,7 +589,7 @@ def configure_authentication(self, authentication_handler: AuthenticationHandler
 
 class SubRouter(Robyn):
     def __init__(self, file_object: str, prefix: str = "", config: Config = Config(), openapi: OpenAPI = OpenAPI()) -> None:
-        super().__init__(file_object, config, openapi)
+        super().__init__(file_object=file_object, config=config, openapi=openapi)
```

Comment:
Hey @VishnuSanal , now that I think of it. Can we remove openapi from the subrouters?

Reply:
> Can we remove openapi from the subrouters?

noooo!! how does endpoints given via `@subrouter.METHOD` work? ref: https://robyn.tech/documentation/example_app/openapi#how-does-it-work-with-subrouters

Reply:
Alright, makes sense.